corehq/apps/custom_data_fields/templates/custom_data_fields/custom_data_fields.html

```diff
@@ -113,9 +116,9 @@
                                         </span>
                                     </th>
                                     <th class="col-sm-3">
-                                        {% trans "Choices" %}
+                                        {% trans "Validation" %}
                                         <span class="hq-help-template"
-                                              data-title="{% trans "Choices" %}"
+                                              data-title="{% trans "Validation" %}"
```

Comment:
Pretty nit picky but this heading might be a little confusing when the only option is "choices" and nothing else shows up. I wonder if we should change the word depending on the toggle?

Reply:
agreed